csi-attacher/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.13.5 AS build
+ENV GIT_UPSTREAM_REPO=https://github.com/kubernetes-csi/external-attacher
+ENV GIT_TAG=v1.2.1
+WORKDIR /go/src/github.com/storageos/images/external-attacher
+COPY LICENSE .
+# Clone upstream and build from target tag directly.
+RUN git clone $GIT_UPSTREAM_REPO $GOPATH/src/github.com/kubernetes-csi/external-attacher
+RUN cd "$GOPATH/src/github.com/kubernetes-csi/external-attacher" \
+    && git checkout $GIT_TAG \
+    && make
```

Comment:
Using `make` directly because the make target builds windows based on a condition, not always.